packages/std/src/results/events.rs

```diff
@@ -121,7 +134,9 @@ mod tests {
             ty: "test".to_string(),
             attributes: vec![attr("foo", "bar"), attr("bar", "baz")],
         };
-        let event_builder = Event::new("test").attr("foo", "bar").attr("bar", "baz");
+        let event_builder = Event::new("test")
```

Comment:
How about using add_attributes here in the test as well to give it some coverage?

Reply:
I don't think you addressed this comment.

Reply:
Sorry, it's there now.